### PR TITLE
feat: added GET endpoints

### PR DESCRIPTION
**Pull request by k-avy**
fixes the first two tasks of issue 1

This consists of two end points:
* Get all licenses
  * to get all the details of the licenses present in the database.
  * gets the data of all the data components present in the database.
 * Get licenses by shortname:
   *  to get the data of the license with the shortname passed as a parameter.
   * error when no such license found.

### CODE REVIEW

File: cmd/laas/main.go [hunk 1]
```diff
@@ -18,16 +21,18 @@ import (
...
+	// port number of the host
 	port = flag.String("port", "5432", "port number")
-	// argument to enter the name of database user
+	// database user
 	user = flag.String("user", "fossy", "user name")
-	// argument to enter the name to database to be connected
+	// name of database to be connected
 	dbname = flag.String("dbname", "fossology", "database name")
-	// argument to enter the password of the user
+	// password of the user
```
Comment (avinal) [inline cmd/laas/main.go:30]:
```suggestion
	// password of the database
```

File: cmd/laas/main.go [hunk 2]
```diff
@@ -18,16 +21,18 @@ import (
...
 	password = flag.String("password", "fossy", "password")
-	// argument to enter the path of data file
+	// path of data file
 	datafile = flag.String("datafile", "licenseRef.json", "datafile path")
+	// boolean agument to whether update the database or not
```
Comment (avinal) [inline cmd/laas/main.go:34]:
```suggestion
	// auto-populate the database
```

**Review by avinal — CHANGES_REQUESTED** [review]
Good work, few suggestions

File: go.mod [hunk 3]
```diff
@@ -3,16 +3,41 @@ module github.com/fossology/LicenseDb
 go 1.18
```
Comment (avinal) [inline go.mod:3]:
```suggestion
go 1.20
```
go 1.18 has already reached EOL, please update your installation and then update this mod file.

**Review by GMishx — COMMENTED** [review]
Changes looks good. Please check the comments from @avinal

**Review by ag4ums — APPROVED** [review]
Changes looks good

**Pull request merged**